tests/tizen_capi/unittest_tizen_capi.cc

```diff
@@ -4814,6 +4815,7 @@ TEST (nnstreamer_capi_singleshot, set_input_info_success_02)
   out_dim[2] = 1;
   out_dim[3] = 1;
   ml_tensors_info_set_count (out_info, 1);
+  ml_tensors_info_set_tensor_name (out_info, 0, "output");
```

Comment:
The test should work without tensor names. If you want to test with tensor names, please add more test cases (and negatives...)